StreamChatClient_v3/Errors/ServerResponseError.swift

```diff
@@ -0,0 +1,33 @@
+//
+// ServerResponseError.swift
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// A parsed server response error.
+struct ServerResponseError: LocalizedError, Decodable, CustomDebugStringConvertible, Error {
```

Comment:
~Why isn't this under `ClientError`?~
I understood a little bit late that this is something our backend sends. So, naming could be improved, such as `ServerErrorResponse`

Reply:
Indeed, the naming of these objects is all over the place. I'm actually working on improving the overall naming and structure of "_Response" objects as a part of CIS-155.